sa/model.go

```diff
@@ -617,3 +617,11 @@ type keyHashModel struct {
 	CertNotAfter time.Time
 	CertSerial   string
 }
+
+type blockedKeyModel struct {
+	ID      int64
+	KeyHash []byte
+	Added   time.Time
+	Source  string
```

Comment:
I think we should use an int instead of a string. Would save space since there's only ever likely to be two sources: API and admin.